lib/plugins/inventory.js

```diff
@@ -469,7 +469,8 @@ function inject (bot, { version, hideErrors }) {
   }
 
   function createActionNumber () {
-    return nextActionNumber++
+    nextActionNumber = nextActionNumber === 32767 ? 1 : nextActionNumber+1
+    return nextActionNumber
```

Comment:
```suggestion
    nextActionNumber = nextActionNumber === 32767 ? 1 : nextActionNumber + 1
    return nextActionNumber - 1
```
fix lint and return the same as previously,

```js
let x = 0;
return x++;
```
the number returned here would be 0, the number before the value was incremented